careerfairsystems/arkad-app
# A company can now remove a student, stil need the backend to finish t…

…o be able to remove student completly

## Change description

A company can now remove a student when the backend is finished. A button has been added which will show a modal.

## How to verify

When the remove button in the student card view is pressed a modal will appear. When pressing close the modal should close and when remove is pressed the modal should close and list view of blipped students should show. 

## Issues fixed

#216
